desktop/desktop/lib/state/README.md

```diff
@@ -0,0 +1,6 @@
+State
```

Comment:
I agree we need a decision here – @spen, do you think we will be able to quickly enough make everything work in the Calypso repo, so that we merge a working version of this and address the rest of the non-blocker feedback (like @mtias’s many suggestions) in subsequent PRs. What’s the minimum amount of changes needed to move completely from the other repo?